test/e2e/app-dir/next-after-app/app/edge/invalid-in-client/page.js

```diff
@@ -0,0 +1,6 @@
+// (replaced in tests)
+// 'use client'
```

Comment:
Is this something we can just create another page for?

Reply:
would need to put it in a whole different app, because otherwise it wont' compile and thus break the `next start` tests

Reply:
moved that and the other one to `test/e2e/app-dir/next-after-app-invalid-usage/`, so they no longer need patching